app/assets/v2/js/amounts.js

```diff
@@ -0,0 +1,31 @@
+var getUSDEstimate = function (amount, denomination) {
+    if (document.conversion_rates && document.conversion_rates[denomination]) {
+        var usd_amount = amount / document.conversion_rates[denomination];
+    } else {
+        var request_url = '/sync/get_amount?amount=' + amount + '&denomination=' + denomination;
+        jQuery.get(request_url, function (result) {
+            var usd_amount = result['usdt'];
+            var conv_rate = amount / usd_amount;
```

Comment:
i think there was a mix up about what this endpoint returns.  it returns `usd_amount` which is the eth/usd conv rate, and `eth_amount` which is the denomination/eth conv rate

this should be `conv_rate = amount / usd_amount / eth_amount`

Reply:
@rajatkapoor Could we replace `var` with `let` / `const` from `ES6` ?

Reply:
Hi.
No we can't do that as this is pure client side JS with no babel transpilers in between. If we use ES6/7 then we will mis function on a lot of browsers which do not have es6/7 implemented

Reply:
Ah I assumed that there was basic support for `let`, `const` & `arrow functions` in almost all the browsers. I usually follow [this](http://kangax.github.io/compat-table/es5/).

Bad idea to assume developers who visit this site -> don't use legacy browsers 😓 

Also is it worth considering integrating babel transpilers ?

Reply:
@owocki I see this endpoint returns two things: 
```
            'eth': amount_in_eth,
            'usdt': amount_in_usdt
```
According to which this code looks fine. Let me know if I am missing something somewhere

Reply:
@thelostone-mc We will be missing a lot on a lot of users on android/webview users. 
For such compatibility check, I use: https://caniuse.com/
Also, moving to es6 will be good idea if we plan to move full fledged. Then, I would also like to pitch in to use typescript in order to make the development process even better :)

Reply:
@rajatkapoor 

the UI still shows the ETH conversion rate even when "EDG" or any ERC20 token is selected in the dropdown.  http://bits.owocki.com/3v1r2I2Q0m2m/Screen%20Shot%202017-12-11%20at%209.29.24%20AM.png

this is incorrect.  you need use the returned responses from the endpoint as i described in https://github.com/gitcoinco/web/pull/84/files#r155141576 to make it work